src/style.css

```diff
@@ -0,0 +1,109 @@
+html {
+  font-family: "Roboto", sans-serif;
+  margin: 0;
+}
+  
+* {
+  box-sizing: border-box;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  padding: 29px 14px 15px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  width: 160px;
+  height: 134px;
+  margin: auto;
+  margin-bottom: 33px;
+}
+
+.card__name {
+  margin-bottom: 6px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.card__code {
+  margin-bottom: 15px;
+  font-size: 11px;
+  line-height: 13px;
```

Comment:
You are repeating 
```
  font-size: 11px;
  line-height: 13px;
```

Three times in your code. You can move it to `body` to make it common font size for your page